.github/workflows/integration-tests.yml

```diff
@@ -2,47 +2,84 @@ name: Integration tests
 
 on:
   workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to run the tests against'
+        type: choice
+        required: true
+        default: 'dev'
+        options:
+          - dev
+          - staging
```

Comment:
I like it. The only downside though is it's not visible which environment the last test run against until you go into the logs.

Reply:
I just added an extra step to keep the info
![image](https://user-images.githubusercontent.com/10419887/208159919-44261905-0a3e-46f5-9b53-7b67958ebe97.png)

Reply:
Awesome! No issues then. :)